frontend/src/shared.js

```diff
@@ -0,0 +1,138 @@
+export const tagsOptions = [
+  {
+    value: 'research',
+    key: 'research',
+    text: 'Research'
+  }, {
+    key: 'contracts',
+    value: 'contracts',
+    text: 'Contracts'
+  }, {
+    key: 'frontend',
+    value: 'frontend',
+    text: 'Frontend'
+  }, {
+    value: 'twitter',
+    key: 'twitter',
+    text: 'Twitter'
+  }, {
+    value: 'solidity',
+    key: 'solidity',
+    text: 'Solidity'
+  }, {
+    key: 'did',
+    value: 'did-token',
+    text: 'DID Token'
+  }, {
+    key: 'hav-token',
+    value: 'hav-token',
+    text: 'HAV Token'
+  }, {
+    key: 'react',
+    value: 'react',
+    text: 'React'
+  }, {
+    key: 'html',
+    value: 'html',
+    text: 'HTML'
+  }, {
+    value: 'frontend-tests',
+    key: 'frontend-tests',
+    text: 'Frontend Tests'
+  }, {
+    value: 'contract-tests',
+    key: 'contract-tests',
+    text: 'Contract Tests'
+  }, {
+    key: 'css',
+    value: 'css',
+    text: 'CSS'
+  }, {
+    key: 'ideas',
+    value: 'ideas',
+    text: 'Ideas'
+  }, {
+    key: 'governance',
+    value: 'governance',
+    text: 'Governance'
+  }, {
+    key: 'code-review',
+    value: 'code-review',
+    text: 'Code Review'
+  }, {
+    key: 'applications',
+    value: 'applications',
+    text: 'Applications'
+  }, {
+    key: 'security',
+    value: 'security',
+    text: 'Security'
+  }, {
+    key: 'social',
+    value: 'social',
+    text: 'Social'
+  }, {
+    key: 'administration',
+    value: 'administration',
+    text: 'Administration'
+  }, {
+    key: 'decisions',
+    value: 'decisions',
+    text: 'Decisions'
+  }, {
+    key: 'design',
+    value: 'design',
+    text: 'Design'
+  }, {
+    key: 'open-source',
+    value: 'open-source',
+    text: 'Open Source'
+  }, {
+    key: 'meetups',
+    value: 'meetups',
+    text: 'Meetups'
+  }, {
+    key: 'education',
+    value: 'education',
+    text: 'Education'
+  }, {
+    key: 'Contributors',
+    value: 'contributors',
+    text: 'Contributors'
+  }, {
+    key: 'Voting Dapp',
+    value: 'voting-dapp',
+    text: 'Voting Dapp'
+  }, {
+    key: 'planning',
+    value: 'planning',
+    text: 'Planning'
+  }, {
+    key: 'task-management',
+    value: 'task-management',
+    text: 'Task Management'
+  }, {
+    key: 'legal',
+    value: 'legal',
+    text: 'Legal'
+  }, {
+    key: 'crowdsale',
+    value: 'crowdsale',
+    text: 'Crowdsale'
+  }
+]
```

Comment:
This could just be an array of the titles, and then we can just slugify the titles when we need them as keys/values.

Reply:
Where are you thinking of doing the slugifying?

Reply:
`componentDidMount`?

Reply:
Can be done whenever you need it when rendering. For the `Dropdown`, you could use a helper function and map the tags to options:
```js
const tagOption = tag => {
  const value = slug(tag)
  return { text: tag, key: value, value }
}

options={tags.map(tagOption)}
```

Reply:
Yeah just put this helper in shared but we could just put it in dropdown too, although that isn't reusable.

Reply:
The helper is specific to the dropdown. If the dropdown were used in multiple places it could be wrapped in its own component along with the helper.